client/src/main/scala/drt/client/components/ArrivalInfo.scala

```diff
@@ -66,18 +66,14 @@ object FeedSourceRow {
 
   case class Props(feedSourceArrival: FeedSourceArrival, airportConfig: AirportConfig) extends UseValueEq
 
-  def feedDisplayName(isCiriumAsPortLive: Boolean, feedSource: FeedSource) = if (isCiriumAsPortLive)
```

Comment:
If we don't use this function anymore, how do we distinguish between a port's live feed and a cirium live feed when it's configured as a port live feed?